DataTableWriter/Writers/DataTableCSVWriter.cs

```diff
@@ -1,9 +1,12 @@
 ﻿using CsvHelper;
 using log4net;
 using System;
+using System.Collections.Generic;
 using System.Data;
 using System.IO;
+using System.Linq;
 using System.Reflection;
+using System.Text;
```

Comment:
Can we revert these three using statements you added? I think these crept in by mistake.